pkgs/applications/graphics/vengi-tools/default.nix

```diff
@@ -46,6 +43,15 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-8rGnW+VtqNJYqUqQDp0yOVIQd7w+cq7PIpqqIQPhkbE=";
   };
 
+  prePatch = lib.optionalString stdenv.hostPlatform.isDarwin ''
+    # Disable code signing on macOS
+    substituteInPlace cmake/macros.cmake --replace-fail "codesign" "true"
+    substituteInPlace cmake/system/apple.cmake --replace-fail "if(APPLE)" "if(false)"
+
+    # calls otool -L on /usr/lib/libSystem.B.dylib and fails because it doesn't exist
+    substituteInPlace cmake/applebundle.cmake --replace-fail 'fixup_bundle("''${TARGET_BUNDLE_DIR}" "" "")' ""
```

Comment:
This is a CMake bug we can hopefully fix for 25.11, but other packages are already working around it, so this seems fine.